src/main/webapp/app/pages/variantRecommendation/FormatPercentage.tsx

```diff
@@ -0,0 +1,19 @@
+export function formatPercentage(value: number) {
```

Comment:
Let's move this function to `src/main/webapp/app/shared/util/utils.tsx` since it doesn't really need to be in it's own file.